.circleci/config.yml

```diff
@@ -58,3 +64,11 @@ workflows:
           requires:
             - build
```

Comment:
Same thing here that you can probably list only `test` because it already requires `build`